cpp/test/CMakeLists.txt

```diff
@@ -32,10 +31,6 @@ IF(ENABLE_GSL)
     Cap_ADD_BOOST_TEST(test_impedance_spectroscopy      )
     Cap_ADD_BOOST_TEST(test_discrete_fourier_transform  )
 ENDIF()
-IF(ENABLE_TASMANIAN)
```

Comment:
OK, I will remove Tasmanian from SetupTPLs.cmake.
Last place it appears is ``test_exact_transient_solution-2.cc``.
Didn't touch that one to avoid merge conflicts.